### PR TITLE
Added support for building Android x86_64 (for ChromeOS).

### PR DESCRIPTION
**Pull request by monroeberg**
Added support for building Android x86_64 (for ChromeOS).

### CODE REVIEW

File: lib/config-android-x86.h [hunk 1]
```diff
@@ -794,7 +794,7 @@
 #define NTLM_WB_FILE "/usr/bin/ntlm_auth"
 
 /* cpu-machine-OS */
-#define OS "arm-unknown-linux-gnu"
+#define OS "x86-unknown-linux-gnu"
```
Comment (davidrogers-unity) [inline lib/config-android-x86.h:797]: I think this should be i686

File: lib/config-android-x86_64.h [hunk 2]
```diff
@@ -0,0 +1,1031 @@
...
+/* #undef NEED_THREAD_SAFE */
+
+/* Define to enable NTLM delegation to winbind's ntlm_auth helper. */
+#define NTLM_WB_ENABLED 1
+
+/* Define absolute filename for winbind's ntlm_auth helper. */
+#define NTLM_WB_FILE "/usr/bin/ntlm_auth"
+
+/* cpu-machine-OS */
+#define OS "x86-unknown-linux-gnu"
```
Comment (davidrogers-unity) [inline lib/config-android-x86_64.h:797]: x86_64 here

**Review by davidrogers-unity — CHANGES_REQUESTED** [review]
Just two small changes but otherwise lgtm

File: lib/config-android-x86.h (hunk 1 above)
Comment (monroeberg) [reply lib/config-android-x86.h]: Fixed.

File: lib/config-android-x86_64.h (hunk 2 above)
Comment (monroeberg) [reply lib/config-android-x86_64.h]: Fixed.

**Review by aurimasc — APPROVED** [review]

**Review by davidrogers-unity — APPROVED** [review]

**Pull request merged**